adafruit_circuitplayground/express.py

```diff
@@ -658,10 +658,41 @@ def stop_tone(self):
         # Stop playing any tones.
         if self._sample is not None and self._sample.playing:
             self._sample.stop()
-            self._sample.deinit()
-            self._sample = None
         self._speaker_enable.value = False
 
+    def play_melody(self, frequencies, durations):
+        """ Play a melody using notes and beats, rests are 0 frequency.
+
+        :param int frequency: The frequency of the tone in Hz
+        :param float duration: The duration of the tone in seconds
+
+        .. image :: ../docs/_static/speaker.jpg
+          :alt: Onboard speaker
+
+        .. code-block:: python
+
+            from adafruit_circuitplayground.express import cpx
+
+            notes = [698,880,988,698,880,988,698,880,988,1319,1175,988,1047,988,784,659,587,659,784,659,
+            698,880,988,698,880,988,698,880,988,1319,1175,988,1047,1319,988,784,988,784,587,659,
+            587,659,698,784,880,988,1047,988,659,698,784,880,988,1047,1175,1319,1397,1568,
+            587,659,698,784,880,988,1047,988,659,
+            784,698,880,784,988,880,1047,988,1175,1047,1319,1175,1397,1319,1319,1397,1175,1319,
+            0,0,0]
+
+            beats= [2,2,4,2,2,4,2,2,2,2,4,2,2,2,2,8,2,2,2,10,
+            2,2,4,2,2,4,2,2,2,2,4,2,2,2,2,8,2,2,2,10,
+            2,2,4,2,2,4,2,2,8,2,2,4,2,2,4,2,2,8,
+            2,2,4,2,2,4,2,2,8,
+            2,2,2,2,2,2,2,2,2,2,2,2,2,2,1,1,2,6,
+            4,2,2]
```

Comment:
pylint is complaining about the line length (>100) on line 676, and several instances of a trailing whitespace. Beyond that, the indenture is wrong for each list; the continuation lines should be indented to the opening bracket at minimum, when the items start on the same line as the `[`. To maximize the line lenghts, i would suggest using this style (which also meets PEP8's 79 char line length):
```python
           notes = [
               698, 880, 988, 698, 880, 988, 698, 880, 988, 1319, 1175, 988, 1047, 988,
               784, 659, 587, 659, 784, 659, 698, 880, 988, 698, 880, 988, 698, 880, 988,
               1319, 1175, 988, 1047, 1319, 988, 784, 988, 784, 587, 659, 587,659,698,784,
               880, 988, 1047, 988, 659, 698, 784, 880, 988, 1047, 1175, 1319, 1397, 1568,
               587, 659, 698, 784, 880, 988, 1047, 988, 659, 784, 698, 880, 784, 988, 880,
               1047, 988, 1175, 1047, 1319, 1175, 1397, 1319, 1319, 1397, 1175, 1319, 0,
               0, 0
           ]

           beats = [
               2, 2, 4, 2, 2, 4, 2, 2, 2, 2, 4, 2, 2, 2, 2, 8, 2, 2, 2, 10,
               2, 2, 4, 2, 2, 4, 2, 2, 2, 2, 4, 2, 2, 2, 2, 8, 2, 2, 2, 10,
               2, 2, 4, 2, 2, 4, 2, 2, 8, 2, 2, 4, 2, 2, 4, 2, 2, 8, 2, 2,
               4, 2, 2, 4, 2, 2, 8, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2,
               2, 1, 1, 2, 6, 4, 2, 2
           ]
```
Oh, and spaces between comma separated values increase readability.